sovrin_client/test/agent/bulldog.py

```diff
@@ -12,7 +12,7 @@
 from sovrin_client.test.agent.test_walleted_agent import TestWalletedAgent
 from sovrin_client.test.conftest import primes
 from sovrin_client.test.helper import TestClient
-from sovrin_client.test.agent.bulldog_helper import bulldogLogger
+from sovrin_client.test.agent.bulldog_helper import bulldogLogger, getBulldogLogger
```

Comment:
I don't think now anyone would be using bulldogLogger (as now you might be using getBulldogLogger), so please remove it if it is unused.